src/main/java/meditracker/command/ViewCommand.java

```diff
@@ -19,11 +25,26 @@
  */
 public class ViewCommand extends Command {
     public static final ArgumentList ARGUMENT_LIST = new ArgumentList(
-            new ListIndexArgument(false)
+            new ListIndexArgument(true),
+            new NameArgument(true),
+            new QuantityArgument(true),
+            new ExpirationDateArgument(true),
+            new RemarksArgument(true)
     );
+
     public static final String HELP_MESSAGE = ArgumentHelper.getHelpMessage(CommandName.VIEW, ARGUMENT_LIST);
     private final Map<ArgumentName, String> parsedArguments;
 
+    /**
+     * Constructs a FindCommand object with the specified arguments.
```

Comment:
Incorrect JavaDoc description, should be `ViewCommand`

Reply:
Resolved in [628bad9](https://github.com/AY2324S2-CS2113T-T09-1/tp/pull/206/commits/628bad9f955193e6c6f2db66dbf40a7ca8a6c58a)